src/stats.c

```diff
@@ -33,6 +34,42 @@ static struct {
     double user;
 } used_cpu;
 
+// log slow command
+#define BUILD_CMD_ITEM(cmd, type, access) #cmd,
+static const char * cmd_table[] = {CMD_DO(BUILD_CMD_ITEM)};
+static const size_t CMD_NUM = sizeof(cmd_table) / sizeof(char*);
+
+static uint32_t *counts_sum;
```

Comment:
CMD_NUM is not a compile time constant, or should I add a tag at the end of [command enum](https://github.com/eleme/corvus/blob/master/src/command.h#L153) to get the number of command?